.github/workflows/build.yml

```
@@ -152,3 +152,49 @@ jobs:
         cmake --version | grep 3.10.3
     - name: Build
       run: script/test package
+
+  ot12-backbone-ci:
```

nit: adding "-ci" seems redundant, since everything in GitHub Actions is related to continuous integration.
```suggestion
  ot12-backbone:
```